recipes.js

```diff
@@ -1,11 +1,65 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js')
+const chalk = require("chalk");
 
-mongoose.connect('mongodb://localhost/recipeApp')
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const data = require("./data.js");
+
+const express = require("express");
+//const app = express;
+//app.set("view engine", "hbs");
+
+const Recipe = require("./recipes-model.js");
+
+mongoose
+  .connect(
+    "mongodb://localhost/recipeApp",
+    { useNewUrlParser: true }
+  )
   .then(() => {
-    console.log('Connected to Mongo!')
-  }).catch(err => {
-    console.error('Error connecting to mongo', err)
+    console.log("Connected to Mongo!");
+  })
+  .catch(err => {
+    console.error("Error connecting to mongo", err);
   });
 
+//------------------------RECIPE CREATE ------------------------
+
+Recipe.create({
+  title: "naan bread",
+  level: "Easy Peasy",
+  cousine: "Indian",
+  dishType: "Dish",
+  duration: 15
+})
+  .then(recipeDoc => {
+    console.log(chalk.yellow(`${recipeDoc.title}`));
```

Comment:
awwww sorry, I just saw now that you did use the chalk package and provided me with a colorful console, you just made my mongoose correction a lot nicer, thank you 😌